aptos-node/Cargo.toml

```diff
@@ -82,4 +82,5 @@ consensus-only-perf-test = ["aptos-executor/consensus-only-perf-test", "aptos-me
 default = []
 failpoints = ["fail/failpoints", "aptos-consensus/failpoints", "aptos-executor/failpoints", "aptos-mempool/failpoints", "aptos-api/failpoints"]
 indexer = ["aptos-indexer"]
+network-perf-test = ["aptos-peer-monitoring-service-client/network-perf-test", "aptos-peer-monitoring-service-server/network-perf-test", "aptos-peer-monitoring-service-types/network-perf-test"]
```

Comment:
basic question: we do expect this feature to be default at some point, right?

Reply:
No, not in a production environment. This feature will only be for end-to-end network performance benchmarking (similar to the consensus only tests). One application is to deploy this feature in a chaos-mesh like environment (the one you're working on), so we can see how latencies and packet drops might affect end-to-end throughput. Does that make sense?